### PR TITLE
Fix aiohttp.server examples to run with the latest version of aiohttp

### PR DESCRIPTION
**Pull request by GMLudo**
Without theses fixes, you have stacktraces to read headers and for 404 and 302 pages.

### CODE REVIEW

File: examples/srv.py [hunk 1]
```diff
@@ -38,15 +38,15 @@ def handle_request(self, message, payload):
                 isdir = os.path.isdir(path)
 
         if not path:
-            raise aiohttp.HttpProcessingError(404)
+            raise aiohttp.HttpProcessingError(code=404)
 
-        for hdr, val in message.headers.items(getall=True):
-            print(hdr, val)
+        for hdr in message.headers:
```
Comment (asvetlov) [inline examples/srv.py]: Why did you change the code from printing all headers to unique-only ones?

**Comment by ludovic-gasc** [comment]
@asvetlov Is it ok now, or I've missed something else ?

**Comment by asvetlov** [comment]
Everything is fine, thanks
